### ISSUE TITLE
Adicionar um icon do git ao lado do titulo do programa

### ISSUE THREAD

**Issue by JoseOda** [JoseOda/TrabalhoGCS#13]
Seria interessante adicionar uma logo do github ao lado do nome do programa

**Comment by JoseOda** [comment]
![image](https://user-images.githubusercontent.com/95934487/175840294-6ca7f883-0dc9-461c-b56c-76ea67a0ec85.png)

**Comment by joaovitormoitinho** [comment]
achei muito boa a ideia para deixar a aplicação mais visível que ela voltada para o gitHub.

**Comment by enzookita** [comment]
Sim, deixaria a página mais amigável ao usuário. @hugor1256, o que você acha a respeito?

**Comment by GabrielCGon** [comment]
Concordo com a ideia.

**Comment by halls2001** [comment]
Concordo irei prosseguir com a implementação
